### ISSUE TITLE
Playground fails to load script module due to incorrect MIME type

### ISSUE THREAD

**Issue by skykanin** [solidjs/solid-playground#161]
Code in the solid playground doesn't render any output. In the console I see this error: `Failed to load module script: Expected a JavaScript module script but the server responded with a MIME type of "application/octet-stream". Strict MIME type checking is enforced for module scripts per HTML spec.`

I was able to reproduce this issue both in Firefox version `117.0.1` and Google Chrome version `117.0.5938.88`.

![image](https://github.com/solidjs/solid-playground/assets/3789764/7bdc2eae-2917-402e-8372-216c720248e1)

**Comment by skykanin** [comment]
Enabling my VPN fixes the issue so I'm guessing this is a DNS issue with https://jspm.dev

**Comment by modderme123** [comment]
One solution would be to change the cdn used by clicking the tab to the right of main.tsx

**Comment by skykanin** [comment]
Yeah, changing the `"solid-js/web"` entry in the import map to `"https://jspm.dev/npm:solid-js/web"` works as a workaround.

**Comment by paganaye** [comment]
My first 5 minutes with the framework did not go well.
Perhaps the playground needs a fallback mechanism.
